tests/pipeline/utils.py

```diff
@@ -308,148 +237,112 @@ def assert_only_table_columns(
 #
 # Load table counts
 #
-def _load_table_counts_fs(p: dlt.Pipeline, *table_names: str) -> DictStrAny:
-    file_tables = _load_tables_to_dicts_fs(p, *table_names)
-    result = {}
-    for table_name, items in file_tables.items():
-        result[table_name] = len(items)
-    return result
 
 
-def _load_table_counts_sql(p: dlt.Pipeline, *table_names: str) -> DictStrAny:
-    with p.sql_client() as c:
-        qualified_names = [c.make_qualified_table_name(name) for name in table_names]
-        query = "\nUNION ALL\n".join(
-            [
-                f"SELECT '{name}' as name, COUNT(1) as c FROM {q_name}"
-                for name, q_name in zip(table_names, qualified_names)
-            ]
-        )
-        with c.execute_query(query) as cur:
-            rows = list(cur.fetchall())
-            return {r[0]: r[1] for r in rows}
+def load_table_counts(p: dlt.Pipeline, *table_names: str) -> DictStrAny:
+    """Returns row counts for `table_names` as dict, if no table names are given, all data tables are counted"""
```

Comment:
learning question (feel free to ignore): does this have to be DictStrAny?
why not dict str int?

Reply:
Yeah, good point, I'm just using it because this type has always been here ;)